leases/utils.py

```diff
@@ -63,3 +63,34 @@ def calculate_berth_lease_end_date() -> date:
     Leases always end on 14.9, the year depends on the current year
     """
     return calculate_season_end_date()
+
+
+def calculate_winter_storage_lease_start_date() -> date:
+    """
+    Return the date when the "winter" lease season should start
+
+    The season should start by default on 15.9 (one day after the "summer" season ends)
+    """
+
+    # Return the earliest date between the default start date or today
+    today = date.today()
+    default = max(date(day=15, month=9, year=today.year), today)
+
+    if today < date(10, 6, today.year):
+        return min(default, today)
```

Comment:
hm, `min()` here is kinda redundant, since `default` cannot be `< date(15, 9, today.year)`. not that it affects anything 🤔

Reply:
🤔 :derp: Good catch actually, doesn't affect, but it's entirely unnecessary.